dashboard/v1.0/src/components/benchmarks/FloodPing.tsx

```diff
@@ -161,13 +161,42 @@ export default class FloodPing extends Component<
       });
   };
 
+  public opts = (operation: string) => {
+    switch (operation) {
+      case 'start':
+        this.connection.signalFloodPingStart().then(res => {
+          if (res.data) {
+            alert('Flood Ping routine started');
+          }
+        });
+        break;
+      case 'stop':
+        this.connection.signalFloodPingStop().then(res => {
+          if (res.data) {
+            alert('Flood Ping routine stopped');
+          }
+        });
+        break;
+    }
+  };
+
```

Comment:
Can we make this a new independent functional component since this is common for every module. Further, this will involve more than `start`, `stop` in the later stage, so we have to write each functionality 4 times. So, better if we can make this into one component.

Reply:
Shall I do this now? , Or we can keep this for new contributors as a good first issue?

Reply:
better to keep this for the new contributors. Please open a new issue and create a new tag `react-ui` and assign it. Also, give the `good first issue` as well.